lib/ecto/multi.ex

```diff
@@ -219,6 +219,17 @@ defmodule Ecto.Multi do
   Adds an insert operation to the multi.
 
   Accepts the same arguments and options as `c:Ecto.Repo.insert/2` does.
+
+  ## Example
+
+      iex> post = %Post{title: "first"}
```

Comment:
This module is doctested, which means we cannot actually leverage `%Post` because there is no guarantee a Post module was defined before.

My suggestion is to remove the doctests (i.e.`iex>`). I would also remove to_list in favor of transaction. Something like:

```elixir
post = %Post{title: "first"}
multi = Ecto.Multi.new |> Ecto.Multi.insert(:insert, post) 
MyApp.Repo.transaction(multi)
```